yt_dlp/extractor/youtube.py

```diff
@@ -3625,12 +3642,14 @@ def _extract_from_tabs(self, item_id, ytcfg, data, tabs):
             })
         if playlist_id is None:
             playlist_id = item_id
+
+        view_count = self._get_count(primary_sidebar_renderer, ['stats', 1])
         if title is None:
-            title = (
-                try_get(data, lambda x: x['header']['hashtagHeaderRenderer']['hashtag']['simpleText'])
-                or playlist_id)
+            title = self._get_text(['header', 'hashtagHeaderRenderer', 'hashtag']) or playlist_id
         title += format_field(selected_tab, 'title', ' - %s')
         title += format_field(selected_tab, 'expandedText', ' - %s')
+
+        last_updated_unix, _ = self._extract_time_text(primary_sidebar_renderer, ['stats', 2])
```

Comment:
wonder if its worth removing time_text being returned. only used in comments in which that isn't supposed to be used anyways.